app.js

```diff
@@ -109,6 +109,20 @@ var ontorrent = function (torrent) {
   var hotswaps = 0
   var verified = 0
   var invalid = 0
+  var downloadedPercentage = 0
+
+  function updateDownloadedPercentage () {
+    var downloaded = 0
+    for (var i = 0; i < engine.torrent.pieces.length; i++) {
+      if (engine.bitfield.get(i)) downloaded++
+    }
+
+    engine.on('download', function () {
```

Comment:
there's no point in subscribing to `download` event, since `downloaded` is a local variable and won't be updated when this function exits